pandas/tests/frame/test_query_eval.py

```diff
@@ -29,6 +29,11 @@ def engine(request):
     return request.param
 
 
+@pytest.fixture(params=[False, True])
```

Comment:
Does this need to be a fixture? I think it would be better served to have a generic True / False fixture instead of duplicating this for particular parameters (if one doesn't already exist)

Reply:
Thank you for your comment. I could not find such a generic boolean fixture in `tests/**/*.py`. How do you think about adding, say, `def boolean` fixture in `tests/frame/common.py`? Am I missing your point?